stages/org.osbuild.rpm

```diff
@@ -379,6 +379,7 @@ def main(tree, inputs, options):
         machine_id_file = pathlib.Path(f"{tree}/etc/machine-id")
         machine_id_file.unlink()
         machine_id_file.touch()
```

Comment:
You can pass `mode=0o444` to touch to avoid the extra step and a race ;)
https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch

Reply:
Thanks, fixed that! :)